cachi2/interface/cli.py

```diff
@@ -432,30 +432,50 @@ def merge_sboms(
 
     first to produce SBOMs to merge.
     """
-    sboms_to_merge: List[Union[Sbom, SPDXSbom]] = []
+    sboms_to_merge: List[Union[SPDXSbom, Sbom]] = []
     for sbom_file in sbom_files_to_merge:
+        sbom_dict = json.loads(sbom_file.read_text())
+        # Remove extra fields which are not in Sbom or SPDXSbom models
+        # Both SBom and SPDXSBom models are only subset of cyclonedx and SPDX specifications
+        # Therefore we need to make sure only fields accepted by the models are present
```

Comment:
Do we need to perform this cleanup? I get that our models are only a subset of the full spec, but since this function is limited to support only Cachi2-produced SBOMs, I believe we could skip this sanitization here. Any SBOM that contains extra fields was not produced by Cachi2, and we don't need to handle them, we can just fail the request.